### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/ACS/acs_subarrays/acs_subarrays.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#170]
## What
An accessibility scan flagged 32 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_subarrays/acs_subarrays.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell3">` (selector: `#codecell3`)
- `<pre id="codecell6"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jd5702jwq_raw.fits to ./mastDownload/HST/jd5702jwq/jd5702jwq_raw.fits ...
</pre>` (selector: `#codecell6`)
- `<pre id="codecell9">` (selector: `#codecell9`)
- `<pre id="codecell10"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w6q_spt.fits to ./mastDownload/HST/j92sa0w6q/j92sa0w6q_spt.fits ...
</pre>` (selector: `#codecell10`)
- `<pre id="codecell12"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w6q_raw.fits to ./mastDownload/HST/j92sa0w6q/j92sa0w6q_raw.fits ...
</pre>` (selector: `#codecell12`)
- `<pre id="codecell14"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w7q_spt.fits to ./mastDownload/HST/j92sa0w7q/j92sa0w7q_spt.fits ...
</pre>` (selector: `#codecell14`)
- `<pre id="codecell16"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w7q_raw.fits to ./mastDownload/HST/j92sa0w7q/j92sa0w7q_raw.fits ...
</pre>` (selector: `#codecell16`)
- `<pre id="codecell18"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w8q_spt.fits to ./mastDownload/HST/j92sa0w8q/j92sa0w8q_spt.fits ...
</pre>` (selector: `#codecell18`)
- `<pre id="codecell20"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w8q_raw.fits to ./mastDownload/HST/j92sa0w8q/j92sa0w8q_raw.fits ...
</pre>` (selector: `#codecell20`)
- `<pre id="codecell22"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w9q_spt.fits to ./mastDownload/HST/j92sa0w9q/j92sa0w9q_spt.fits ...
</pre>` (selector: `#codecell22`)
- `<pre id="codecell24"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0w9q_raw.fits to ./mastDownload/HST/j92sa0w9q/j92sa0w9q_raw.fits ...
</pre>` (selector: `#codecell24`)
- `<pre id="codecell26"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j92sa0010_spt.fits to ./mastDownload/HST/j92sa0010/j92sa0010_spt.fits ...
</pre>` (selector: `#codecell26`)
- `<pre id="codecell31">` (selector: `#codecell31`)
- `<pre id="codecell37">` (selector: `#codecell37`)
- `<pre id="codecell38">` (selector: `#codecell38`)
- `<pre id="codecell39">` (selector: `#codecell39`)
- `<pre id="codecell44"><span></span>WARNING:acs_destripe:clean_streak - Maximum number of clipping iterations specified by the user (15) has been reached.
</pre>` (selector: `#codecell44`)
- `<pre id="codecell45"><span></span>INFO:acs_destripe:perform_correction - =====  Overall statistics for de-stripe corrections:  =====
</pre>` (selector: `#codecell45`)
- `<pre id="codecell46"><span></span>INFO:acs_destripe:perform_correction - STDDEV of applied de-stripe corrections 0.952.
</pre>` (selector: `#codecell46`)
- `<pre id="codecell49"><span></span>INFO:acs_destripe:perform_correction - Effective number of clipping iterations: 15.
</pre>` (selector: `#codecell49`)
- `<pre id="codecell50"><span></span>INFO:acs_destripe:perform_correction - Effective number of additional (repeated) cleanings: 0.
</pre>` (selector: `#codecell50`)
- `<pre id="codecell53">` (selector: `#codecell53`)
- `<pre id="codecell56">` (selector: `#codecell56`)
- `<pre id="codecell58">` (selector: `#codecell58`)
- `<pre id="codecell59">` (selector: `#codecell59`)
- `<pre id="codecell60">` (selector: `#codecell60`)
- `<pre id="codecell70">` (selector: `#codecell70`)
- `<pre id="codecell71">` (selector: `#codecell71`)
- `<pre id="codecell76"><span></span>[&apos;CCDAMP&apos;, &apos;CCDCHIP&apos;, &apos;BINX&apos;, &apos;BINY&apos;, &apos;NX&apos;, &apos;NY&apos;, &apos;TRIMX1&apos;, &apos;TRIMX2&apos;, &apos;TRIMY1&apos;, &apos;TRIMY2&apos;, &apos;BIASSECTA1&apos;, &apos;BIASSECTA2&apos;, &apos;BIASSECTB1&apos;, &apos;BIASSECTB2&apos;, &apos;DESCRIPTION&apos;, &apos;VX1&apos;, &apos;VX2&apos;, &apos;VY1&apos;, &apos;VY2&apos;]
</pre>` (selector: `#codecell76`)
- `<table class="table">` (selector: `#update-oscntab > .pst-scrollable-table-container > table`)
- `<pre id="codecell88">` (selector: `#codecell88`)
- `<pre id="codecell89">` (selector: `#codecell89`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
